mkdocs.yml

```diff
@@ -5,44 +5,44 @@ nav:
       - Introduction: introduction.md
       - Common principles: common-principles.md
       - Modality agnostic files: modality-agnostic-files.md
-      - Modality specific files:
-        - Magnetic Resonance Imaging: modality-specific-files/magnetic-resonance-imaging-data.md
-        - Magnetoencephalography: modality-specific-files/magnetoencephalography.md
-        - Electroencephalography: modality-specific-files/electroencephalography.md
-        - Intracranial Electroencephalography: modality-specific-files/intracranial-electroencephalography.md
-        - Task events: modality-specific-files/task-events.md
-        - Physiological and other continuous recordings: modality-specific-files/physiological-and-other-continuous-recordings.md
+      - Modality specific files: # keep sections in sorted order when adding/editing
         - Behavioral experiments (with no neural recordings): modality-specific-files/behavioral-experiments.md
```

Comment:
Why do we want these sorted alphabetically? I agree that order of addition to the spec is not ideal, but this feels unintuitive. I would expect EEG/MEG/iEEG/NIRS to be near each other, for example.

Reply:
In case of ambiguity I prefer "explicitness" of sorted order.  It has been already many times I had to search up for a section because I managed to miss it in some "intuitive" order. 
If there is no "flow" through the sections requiring some to precede another (is there?) I would really prefer explicit alphabetic sorting even if it would not have some "intuitive feel" ;)

Reply:
My intuitive feel was different and actually close to the proposal of @yarikoptic ... The first time I rendered the doc for the BEP020 I though my modifications didn't work as i couldn't find it in the top of the list ("Eyetracking" starting with "E")... this is why i changed it first, before being noticed that it couldn't be part of my PR.